tests/feature_tests/remediation/remediation.sh

```diff
@@ -5,8 +5,8 @@ METAL3_DIR="$(dirname "$(readlink -f "${0}")")/../../.."
 # shellcheck disable=SC1091
 # shellcheck disable=SC1090
 # shellcheck disable=SC2046
-source "${METAL3_DIR}/scripts/feature_tests/feature_test_vars.sh"
+source "../feature_test_vars.sh"
```

Comment:
Please keep it as absolute path. Then we don't have to worry if someone runs it from an unexpected folder.